config/config.sample.pythnet.toml

```diff
@@ -0,0 +1,39 @@
+[pythd_api_server]
+listen_address = "127.0.0.1:8910"
+
+[primary_network]
+
+# HTTP(S) endpoint of the RPC node. Public RPC endpoints are usually rate-limited, so a private
+# endpoint should be used in most cases.
+rpc_url = "https://pythnet.rpcpool.com"
+
+# WS(S) endpoint of the RRC node. This is used to subscribe to account changes on the network.
+# This can be omitted when oracle.subscriber_enabled is set to false.
+wss_url = "wss://pythnet.rpcpool.com"
+
+# Path to the key store.
+key_store.root_path = "/path/to/keystore"
+
+# Duration of the interval at which to publish updates
+exporter.publish_interval_duration = "400ms"
+
+# Configuration for the optional secondary network this agent will publish data to.
+# In most cases this should be a Solana endpoint.
+[secondary_network]
+
+# Please use other endpoints as these are rate limited
+rpc_url = "https://api.mainnet-beta.solana.com"
+wss_url = "wss://api.mainnet-beta.solana.com"
+
```

Comment:
i take it the default exporter interval is 1s so that's why it doesn't need to be here?

Reply:
yes that's right. I'll add the default value so it be more clear for people.